src/Microsoft.Win32.Primitives/tests/Microsoft.Win32.Primitives.Tests.builds

```diff
@@ -8,6 +8,10 @@
     <Project Include="Microsoft.Win32.Primitives.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
     </Project>
+    <Project Include="Microsoft.Win32.Primitives.Tests.csproj">
+      <TargetGroup>net46</TargetGroup>
```

Comment:
Is there any reason that we need to build this library unique for this configuration? Can we get away with just running on that target by using the portable build of the test library?